FillableFieldworkAIDriver.lua

```diff
@@ -235,3 +235,12 @@ end
 function FillableFieldworkAIDriver:getFillLevelInfoText()
 	return 'NEEDS_REFILLING'
 end
+
+function FillableFieldworkAIDriver:setLightsMask(vehicle)
+	local x,y,z = getWorldTranslation(vehicle.rootNode);
```

Comment:
Do we really want to use the position to update the lights or would be better to use the state?

Reply:
I believe the just checking the state was turning off lights when still in field, but going for the refill course. This was just a way to keep the lights on until away from crops and machines. If that's not a big deal, I can remove the field check. It just might go into normal headlights a bit early.

I can test again today to make sure that was the case.

Reply:
Confirmed in my testing. Refill state is set while still in the field.